src/main/java/nl/ordina/jobcrawler/scrapers/YachtVacancyScraper.java

```diff
@@ -117,9 +116,9 @@ protected void setVacancyAbout(Document doc, Vacancy vacancy) {
         // Extracts the about part from the vacancy which starts from the first h2 tag to the second h2 tag.
         Elements aboutElements = doc.select(".description h2 ~ *");
         StringBuilder about = new StringBuilder();
-        for (Element aboutElement : aboutElements) {
+        for(Element aboutElement : aboutElements) {
             // If the tagName() is h2 it means we reached the end of the 'about' part.
-            if ("h2".equals(aboutElement.tagName()))
+            if("h2".equals(aboutElement.tagName()))
```

Comment:
It's good to see this construct. A String can be compared to null, but if you'd compare null to a String, then you'd get a NullPointerException.